### ISSUE TITLE
Create a redial function

### ISSUE THREAD

**Issue by visola** [visola/go-http-cli#36]
After #34 is done, it would be nice to have a way to search and re-execute requests. An initial idea is to use something like https://github.com/gizak/termui to create a simple UI that can be used to search the history and re-execute previous requests.

**Comment by visola** [comment]
Another possibility is [tview](https://github.com/rivo/tview).
